src/ResourceManager/Network/ChangeLog.md

```diff
@@ -4,6 +4,7 @@
     Changes for the current release should go under the section titled "Current Release", and should adhere to the following format:
 
     ## Current Release
+    * Add DisableBgpRoutePropagation property in RouteTable and effectiveRoute.
```

Comment:
@neaggarwMS oops, I meant the `## Current Release` down below, not in this commented-out section ☺️